client/src/Components/Auth/styles.js

```diff
@@ -0,0 +1,37 @@
+import { makeStyles } from '@material-ui/core/styles';
+
+export default makeStyles((theme) => ({
+  paper: {
+    marginTop: theme.spacing(4),
+    display: 'flex',
+    flexDirection: 'column',
+    alignItems: 'center',
+    padding: theme.spacing(2),
+  },
+  root: {
+    '& .MuiTextField-root': {
+      margin: theme.spacing(1),
+    },
+  },
+  avatar: {
+    margin: theme.spacing(1),
+    backgroundColor: theme.palette.secondary.main,
+  },
+  form: {
+    width: '100%', // Fix IE 11 issue.
+    marginTop: theme.spacing(3),
+  },
+  submit: {
+    margin: theme.spacing(3, 0, 2),
+  },
+    fileInput: {
+    width: '97%',
+    margin: '10px 10px',
+  },
+  profilePic: {
+    margin: '10px 10px',
+    fontWeight: 700,
+    color: "#4B4A4A",
```

Comment:
doesn't have to happen now, but if you use a `.scss` file then you can set variables that are used within your css classes.

This line would then look something like:

`color: $primaryColor`